packages/zeplin-extension-style-kit/declarations/margin.js

```diff
@@ -0,0 +1,36 @@
+import { STYLE_PROPS } from "../constants";
+import Length from "../values/length";
+
+const useRemUnitForMeasurement = ({ useForMeasurements }) => useForMeasurements;
+
+class Margin {
+    static get Zero() {
+        return new Margin(
+            Length.toFourDirectionLengths(
+                { top: 0, right: 0, bottom: 0, left: 0 },
+                { useRemUnitForMeasurement }
```

Comment:
The name of this option doesn't match what the `Length` constructor expects, right?